doc/travis-ci.md

```diff
@@ -113,26 +133,29 @@ For more info, see:
 
 ### `transcrypt`-ed repository
 
-If your repository is `transcrypt`-ed, and you want to access the secrets in Travis CI, then follow these steps:
+If your repository is `transcrypt`-ed, and you want to access the secrets in Travis CI, 
+you need to add `TRANSCRYPT_PASSWORD` variable in Travis Web UI.
 
-```shell
-cd path/to/repo
-travis encrypt "TRANSCRYPT_PASSWORD=<password>"
-```
+The decryption of the repository will happen automatically in non-pull-request builds,
+if `.travis.yml` runs `./travis.sh before_install` in `before_install`
+(default in the [`.travis.yml` template](../repo/dot.travis.yml); see [actual command](../repo/dot.travis.sh)).
 
-Now you can add this to your `.travis.yml` file:
 
-```yaml
-env:
-  global:
-    # TRANSCRYPT_PASSWORD
-    - secure: "..."
+### Migrating secrets from existing repo
+
+To make the setup faster and avoid looking for secrets all over different places, 
+you can _manually_ get all the secrets in existing repo. To do so,
+start a debugging session in Travis UI and after some minutes in Travis output log
```

Comment:
Maybe just link to the section describing how to start a debug build?

Reply:
I will add that, but for a one-time thing it's fine to use Web UI, no need to go and look for the token, read args for the cmd. Easier for me, at least.

Reply:
Oh yeah, definitely. I don't know how the docs look like now but I agree that the docs around "debug build" should prio the UI. The reason I changed my mind is that Travis, at least so far, just works. I guess I'm on the advanced usage, and I don't debug more than once a week on average. In the beginning I thought debugging would happen quite often, thus the travis-debug script... 🙄

Reply:
moved your debug-in-ui docs 👍 https://github.com/tobiipro/support-firecloud/commit/2daade6be15cc046ca1e7bca2a8ed2256fa1ec91